rest_invitations/serializers.py

```diff
@@ -16,69 +16,104 @@
     "email_in_use": _("An active user is using this e-mail address"),
 }
 
+
 class EmailListField(serializers.ListField):
+    """Email list field."""
+
     child = serializers.EmailField()
 
 
 class InvitationWriteSerializer(serializers.ModelSerializer):
-    class Meta:
+    """Invitation write serializer."""
+
+    class Meta(object):
+        """Options."""
```

Comment:
http://legacy.python.org/dev/peps/pep-0008/#documentation-strings

> Write docstrings for all public modules, functions, classes, and methods.

Reply:
@barseghyanartur I see, but this kind of docstrings doesn't add a real documentation content, e.g. Model Meta is a well known structure/pattern in the django ecosystem :), no need to document it.

Reply:
So, you want to have it removed?

Reply:
Yes, but...why did you closed the PR? :)

Reply:
Because I think you're being unreasonably too picky about changes I made. 

If it would be a project with tests, established code style, big community - I would consider making changes. For now, this is a (good) start, with status alpha. And not more than that. 

I had plans submitting more pull requests (tests, for instance). But now I already imagine how much of my time would be eaten up in such discussions.

That is to say, it's not easy to work with you. Therefore, I'm no longer interested in contributing to this project.

No offence and good luck.

Reply:
@barseghyanartur Ok, just to know...as you wish.
Yes, it's an app in alpha status, i accept all valid contributions, but i can't understand why i can't dedicate time to make an accurate review, even on simple changes, pull request are about discussion also :).

BTW, I appreciated the time you dedicated to make your changes,

Have a nice day